tools/pipelines/test-stability.yml

```diff
@@ -0,0 +1,241 @@
+  - Test87
+  - Test88
+  - Test89
+  - Test90
+  - Test91
+  - Test92
+  - Test93
+  - Test94
+  - Test95
+  - Test96
+  - Test97
+  - Test98
+  - Test99
+  - Test100
+
+- name: buildDirectory
+  type: string
+  default: .
+
+- name: taskBuild
+  type: string
+  default: ci:build
+
+- name: taskBuildDocs
+  type: boolean
+  default: false
+
+- name: taskLint
+  type: boolean
+  default: false
+
+- name: taskTest
+  type: object
+  default:
+  - ci:test:mocha
+  - ci:test:jest
+  - ci:test:realsvc:local
+  - ci:test:realsvc:tinylicious
+  - ci:test:stress:tinylicious
+  - test:copyresults
+
+- name: checkoutSubmodules
+  type: boolean
+  default: true
+
+- name: namespace
+  type: boolean
+  default: true
+
+- name: buildNumberInPatch
+  type: string
+  default: false
+
+- name: publishOverride
+  displayName: Publish Override (default = based on branch)
+  type: string
+  default: default
+  values:
+    - default
+    - skip
+    - force
+
+- name: buildToolsVersionToInstall
+  displayName: Fluid build tools version (default = installs version in repo)
+  type: string
+  default: repo
+
+- name: nonScopedPackages
+  displayName: Non-scoped packages to publish
+  type: object
+  default:
+  - fluid-framework
+
+- name: releaseBuildOverride
+  displayName: Release Build (default = not released)
+  type: string
+  default: none
+  values:
+    - none
+    - prerelease
+    - release
+
+- name: tagName
+  type: string
+  default: TestStability
+
+- name: includeInternalVersions
+  type: boolean
+  default: false
+
+schedules:
+  - cron: "00 03 * * SUN"
+    displayName: Scheduled Tests weekly at Saturday night
+    branches:
+      include:
+      - main
+      - next
+    always: true
+
+variables:
+- group: prague-key-vault
+  # We use 'chalk' to colorize output, which auto-detects color support in the
+  # running terminal.  The log output shown in Azure DevOps job runs only has
+  # basic ANSI color support though, so force that in the pipeline
+- name: FORCE_COLOR
+  value: 1
+- template: templates/include-vars.yml
+  parameters:
+    publishOverride: ${{ parameters.publishOverride }}
+    releaseBuildOverride: ${{ parameters.releaseBuildOverride }}
+    buildNumberInPatch: ${{ parameters.buildNumberInPatch }}
+    nonScopedPackages: ${{ parameters.nonScopedPackages }}
+
+stages:
+  - ${{ each stageName in parameters.stageNames }}:
+    - stage:
+      displayName: ${{ stageName }}
+      dependsOn: []
+      jobs:
+      - template: templates/include-test-stability.yml
+        parameters:
+          publishOverride: ${{ parameters.publishOverride }}
+          releaseBuildOverride: ${{ parameters.releaseBuildOverride }}
+          nonScopedPackages: ${{ parameters.nonScopedPackages }}
+          buildToolsVersionToInstall: ${{ parameters.buildToolsVersionToInstall }}
+          buildDirectory: .
+          tagName: ${{ parameters.tagName }}
+          poolBuild: Large
+          checkoutSubmodules: true
+          timeoutInMinutes: 1440
```

Comment:
Should be lower than this with parallelization. Something like 30min * 100 jobs / 20 jobs at once * 2 runs for main & next = 300 min. Plus some safety margin in case it's slower or there's other pipelines running. Don't want to bother consuming all the resources if clearly something is wrong.